specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json

```diff
@@ -534,6 +567,115 @@
       }
     }
   },
+  "ManagedRuleSets": {
```

Comment:
Please put these definitions (this and below until `parameters`) under the [definitions](https://github.com/Azure/azure-rest-api-specs/pull/5921/files#diff-a369a92061eba6d5f1ece36a6c9309a8R264) entry.

You could use [these validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) to check for structure issues locally.

Reply:
Added it on to definitions.